massgov/mayflower
# DP-9567 Remove raw filter from _patterns/01-atoms/06-rich-text/icon-list.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from _patterns/01-atoms/06-rich-text/icon-list.twig.

Use of the template in Mayflower:
```
 ~/MassIT/mayflower/patternlab/ [feature/DP-9567_atom-rich-text-icon-list] grep -r 'icon-list.twig' styleguide/source/_patterns
styleguide/source/_patterns/03-organisms/by-author/rich-text.json:      "path": "@atoms/06-rich-text/icon-list.twig",
styleguide/source/_patterns/05-pages/guide.json:            "path": "@atoms/06-rich-text/icon-list.twig",
styleguide/source/_patterns/05-pages/guide.json:            "path": "@atoms/06-rich-text/icon-list.twig",
styleguide/source/_patterns/05-pages/guide.json:            "path": "@atoms/06-rich-text/icon-list.twig",
 ~/MassIT/mayflower/patternlab/ [feature/DP-9567_atom-rich-text-icon-list]
```

## Related Issue / Ticket

- [DP-9567](https://jira.mass.gov/browse/DP-9567)

## Steps to Test


1. Go to Guide page and check for any display issue with the unordered list with bullet icon.

## Screenshots
<img width="719" alt="dp-9567_mf-sample" src="https://user-images.githubusercontent.com/9633303/43486222-39fc5244-94e1-11e8-90e7-e941cff0865f.png">

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
